cloudferrylib/os/identity/keystone.py

```diff
@@ -260,12 +260,14 @@ def get_endpoint_by_service_type(self, service_type, endpoint_type):
 
         return self.keystone_client.service_catalog.url_for(**kwargs)
 
-    def get_tenants_func(self):
+    def get_tenants_func(self, return_default_tenant=True):
+        default_tenant = self.config.identity.default_tenant \
```

Comment:
This simplifies configuration and reduces chances of having this value set incorrectly. Also user configured in `self.config.cloud.tenant` always exists for any migration type, and it would be safe to use it as default.